addons/web/controllers/main.py

```diff
@@ -708,7 +708,7 @@ def backup(self, master_pwd, name, backup_format = 'zip'):
     def restore(self, master_pwd, backup_file, name, copy=False):
         try:
             data = base64.b64encode(backup_file.read())
-            request.session.proxy("db").restore(master_pwd, name, data, copy)
+            request.session.proxy("db").restore(master_pwd, name, data, bool(copy))
```

Comment:
or we could add str2bool in tools.misc and use it here as well as in odoo/internal#87?